fs/accounting/stats.go

```diff
@@ -63,6 +65,38 @@ Returns the following values:
 ` + "```" + `
 Values for "transferring", "checking" and "lastError" are only assigned if data is available.
 The value for "eta" is null if an eta cannot be determined.
+`,
+	})
+
+	rc.Add(rc.Call{
+		Path:  "core/transferred",
+		Fn:    Stats.Transferred,
+		Title: "Returns stats about completed transfers.",
+		Help: `
+This returns all available stats
+
+	rclone rc core/transferred
+
+Returns the following values:
+
+` + "```" + `
+{
+	"transferred":  an array of completed transfers (including failed ones):
+		[
+			{
+				"name": name of the file,
+				"size": size of the file in bytes,
+				"bytes": total transferred bytes for this file,
+				"checked": if the transfer is only checked (skipped, deleted),
```

Comment:
I'd suggest renaming that to `checked_only` for clarity, technically every file is checked...

Reply:
Or dropping that, we can figure out that is was checked only if `bytes = 0` and `error = ""`, thoughts?